client/package.json

```diff
@@ -13,6 +13,7 @@
     "redux": "^3.7.2"
   },
   "devDependencies": {
+    "jest": "^22.4.3",
```

Comment:
create-react-app already installs jest as it's dependency, but it is the older v20 jest.
I'm in favour of using newer jest but I'm not sure if `react-scripts` will pick it up.
Could you run `react-scripts test -v` to check which jest is used?

Reply:
Yes. I totally forgot about that. No it still will use the old one. If we need the new one, I will edit the npm test script to run the devDep jest (newer version) instead.

Reply:
I think keep v20. react-scripts 2 already includes jest 22. When it is released we could upgrade if we want. What do you think?

Reply:
Yes. Less custom config = less confusion. I will just uninstall the jest v22 from client.